pkg/cmd/roachtest/tests/sysbench.go

```diff
@@ -259,6 +261,31 @@ func runSysbench(ctx context.Context, t test.Test, c cluster.Cluster, opts sysbe
 			return err
 		}
 
+		t.Status("running 75 second workload to collect profiles")
+		{
+			// Create a wait group to wait for the profile collection to finish
+			wg := sync.WaitGroup{}
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				opts := opts
+				opts.duration = 75 * time.Second
+				result, err := c.RunWithDetailsSingleNode(ctx, t.L(), option.WithNodes(c.WorkloadNode()),
+					opts.cmd(useHAProxy)+" run")
+
+				if msg, crashed := detectSysbenchCrash(result); crashed {
+					t.Skipf("%s; skipping test", msg)
+				}
+				require.NoError(t, err)
+			}()
+
+			// Wait for 15 seconds to give a chance to the workload to start, and then
+			// collect CPU, mutex diffs, allocs diffs profiles.
+			time.Sleep(15 * time.Second)
```

Comment:
We have 75s of workload though, and we're only taking a 30s CPU profile, right? I would start the CPU profile at around (shortly after) second 30 then, to get closer to steady state, meaning a 30s sleep here.